front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/Widgets/Selection/Settings/AdminPublicationSearchInput.tsx

```diff
@@ -90,7 +88,7 @@ const AdminPublicationSearchInput = ({
         }}
         onChange={onChange}
         closeMenuOnScroll={false}
-        closeMenuOnSelect={false}
+        closeMenuOnSelect
```

Comment:
@luucvanderzee just quickly assigning you for review to check if this should stay (was intended).